app/assets/javascripts/analytics/_virtualPageViews.js

```diff
@@ -36,20 +36,23 @@
 
   };
 
-  VirtualPageViews.prototype.trackFilterAnalytics = function( group, key, value ) {
+  VirtualPageViews.prototype.trackFilterAnalytics = function( framework, lot, group, key, value ) {
 
-    var filter_path = '/' + group + '/' + key + '/' + value;
-    var url = '/g-cloud/filters' + filter_path;
-    this.sendVirtualPageView( url, 'Filter' + filter_path.replace(/[/]/g, " - ") );
+    var url = '/' + framework + '/' + lot + '/filters' + '/' + group + '/' + key + '/' + value;
+    this.sendVirtualPageView( url, 'Filter' + url.replace(/[/]/g, " - ") );
 
   };
 
   VirtualPageViews.prototype.filterSelected = function(){
     if(this.checked) {
+      var framework = $(this).closest('.options-container').attr('data-framework');
+      framework = framework ? framework : 'g-cloud';
```

Comment:
why does it default to gcloud?

Reply:
@CrystalPea  good point actually, I had initially put this in as g-cloud was the only framework using the feature.